packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx

```diff
@@ -369,6 +450,15 @@ const _SearchInternal = (
                 </ul>
               </div>
             )}
+            <div
+              aria-live="polite"
+              aria-atomic="true"
+              data-testid="search-live-region"
+              className={styles['SubdomainNavBar-search-result-item-suggestion']}
```

Comment:
Nice! As this seems to be visually hidden, can we use that global util class here instead?

```suggestion
              className="visually-hidden"
```

Reply:
This is perfect, thank you for the suggestion!